pandas/io/html.py

```diff
@@ -197,29 +199,32 @@ def parse_tables(self):
         tables = self._parse_tables(self._build_doc(), self.match, self.attrs)
         return (self._build_table(table) for table in tables)
 
-    def _parse_raw_data(self, rows):
-        """Parse the raw data into a list of lists.
+    def _parse_tables(self, doc, match, attrs):
+        """Return all tables from the parsed DOM.
 
         Parameters
         ----------
-        rows : iterable of node-like
-            A list of row elements.
+        doc : tree-like
```

Comment:
As tree-like is not a Python data-type, we should re-specify this.

Reply:
@TomAugspurger : If you think "node-like" or "tree-like" is OK for this documentation, then we can ignore this comment.  However, IMO we should have proper data types in the doc-strings.